frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card.tsx

```diff
@@ -1,168 +1,88 @@
-    body = (
-      <>
-        <div className="skeleton-text nb-data-consumption-card__chart-skeleton" />
-        <GraphEmpty height={200} loading />
-        <div className="skeleton-text nb-data-consumption-card__chart-legend-skeleton" />
-      </>
-    );
-  } else if (!error && !chartData.some(_.isEmpty)) {
-    body = (
-      <>
-        <div className="nb-data-consumption-card__chart-label text-secondary">
-          {CHART_LABELS[sortByKpi]} {suffixLabel}
-        </div>
-        <Chart
-          containerComponent={
-            <ChartVoronoiContainer
-              labelComponent={<ChartTooltip style={{ fontSize: 8, paddingBottom: 0 }} />}
-              labels={({ datum }) => `${datum.y} ${maxUnit}`}
-              voronoiDimension="x"
-            />
-          }
-          minDomain={{ y: 0 }}
-          maxDomain={{ y: maxVal + Math.round(maxVal * 0.25) }}
-          domainPadding={{ x: [padding, padding] }}
-          legendComponent={
-            <ChartLegend
-              themeColor={ChartThemeColor.purple}
-              data={legendData}
-              orientation="horizontal"
-              symbolSpacer={5}
-              gutter={2}
-              height={30}
-              padding={{ top: 50, bottom: 0 }}
-              style={{ labels: { fontSize: 8 } }}
-            />
-          }
-          legendPosition="bottom-left"
-          padding={{
-            bottom: 50,
-            left: 30,
-            right: 20,
-            top: 30,
-          }}
-          themeColor={ChartThemeColor.purple}
-        >
-          <ChartAxis style={{ tickLabels: { fontSize: 8, padding: 2 } }} />
-          <ChartAxis
-            dependentAxis
-            showGrid
-            tickCount={10}
-            style={{
-              tickLabels: { fontSize: 8, padding: 0 },
-            }}
-          />
-          <ChartGroup offset={sortByKpi === BY_EGRESS ? 0 : 11}>
-            {chartData.map((data, i) => (
-              <ChartBar key={`chartbar-${i}`} data={data} /> // eslint-disable-line react/no-array-index-key
-            ))}
-          </ChartGroup>
-        </Chart>
-      </>
-    );
-  } else {
-    body = <GraphEmpty />;
-  }
+  const [data, loading, loadError] = usePrometheusQueries<PrometheusResponse | DataPoint[]>(
+    queries,
+    parser,
+    null,
+    timeSpan[serviceType],
+  );
 
   return (
     <DashboardCard>
       <DashboardCardHeader>
-        <DashboardCardTitle>Data Consumption</DashboardCardTitle>
+        <DashboardCardTitle>Performance</DashboardCardTitle>
         <DataConsumptionDropdown
-          type={metricType}
-          setType={setMetricType}
-          kpi={sortByKpi}
-          setKpi={setsortByKpi}
+          selectedService={serviceType}
+          setSelectedService={setServiceType}
+          selectedBreakdown={breakdownBy}
+          setSelectedBreakdown={setBreakdownBy}
+          selectedMetric={metric}
+          setSelectedMetric={setMetric}
+          isRgwSupported
```

Comment:
@anmolsachan we need to determine where we are going to support it and where not.